src/views/assets/Platform/PlatformCreateUpdate.vue

```diff
@@ -60,23 +60,20 @@ export default {
       url: `/api/v1/assets/platforms/`,
       cleanFormValue: (values) => {
         const protocols = values['protocols'] || []
-        const query = this.$route.query || {}
         const automation = values['automation'] || {}
         const category_type = values['category_type']
         const ansibleConfig = automation?.['ansible_config'] || {}
         automation.ansible_config = ansibleConfig instanceof Object ? ansibleConfig : JSON.parse(ansibleConfig)
 
-        if (query.hasOwnProperty('_clone_from')) {
-          if (automation.hasOwnProperty('id')) {
-            delete automation['id']
-          }
-          values['protocols'] = protocols.map(i => {
-            if (i.hasOwnProperty('id')) {
-              delete i['id']
-            }
-            return i
-          })
+        if (automation.hasOwnProperty('id')) {
+          delete automation['id']
         }
+        values['protocols'] = protocols.map(i => {
+          if (i.hasOwnProperty('id')) {
+            delete i['id']
+          }
+          return i
+        })
         values['category'] = category_type[0]
         values['type'] = category_type[1]
         return values
```

Comment:
There does not appear to be any significant changes that require adjustments in this snippet of code. However, it's essential to always consider the context and use cases when applying these rules. If there is anything specific you want checked or optimized, please specify!